### ISSUE TITLE
pnpm complains of unmet peer dependency.

### ISSUE THREAD

**Issue by CxRes** [veritem/eslint-plugin-vitest#477]
I get the following warning for this package:

``` 
WARN  Issues with peer dependencies found
.
└─┬ eslint-plugin-vitest 0.5.4
  └─┬ @typescript-eslint/utils 7.14.1
    └── ✕ unmet peer eslint@^8.56.0: found 9.5.0
```

Maybe consider placing `@typescript-eslint/utils` an `optionalDependencies`, so we don't even bother with that when not using TS.

**Comment by CxRes** [comment]
Unfortunately, the PR does not solve this issue. `@typescript-eslint` projects suggest moving to `utils` v8-alpha. See the issue I created here:
https://github.com/typescript-eslint/typescript-eslint/issues/9488

**Comment by CxRes** [comment]
Ok! Thanks for reopening and keeping track...

**Comment by Byron2016** [comment]
Same problem 

```console
 WARN  Issues with peer dependencies found

.
└─┬ eslint-plugin-vitest 0.5.4
  └─┬ @typescript-eslint/utils 7.16.0
    └── ✕ unmet peer eslint@^8.56.0: found 9.7.0
```

**Comment by veritem** [comment]
this will be fixed once #479 is merged

**Comment by veritem** [comment]
this will be fixed in the next release
